test/benchmark/benchmark_malloc_realloc_no_memory_tools.cpp

```diff
@@ -0,0 +1,91 @@
+// Copyright 2020 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <benchmark/benchmark.h>
+#include <cstdlib>
+#include <string>
+
+#include "./macros.h"
+
+using BenchmarkFixture = ::benchmark::Fixture;
```

Comment:
This seems like a roundabout way to essentially use no fixture at all. The standard way looks like this: https://github.com/google/benchmark#basic-usage

Unless there is a specific reason you're doing it like this?

Reply:
I guess just too much copy and paste. Adjusted.